docs/syntax.md

```diff
@@ -126,20 +124,25 @@ const GoogleMaps = TSXAir(()=>{
 });
 ```
 
-### afterUpdate
+### afterDomUpdate
 ```tsx
 const InfiniteMeasure = TSXAir(() => {
     const state = store({area:0, ref:{} as RefHolder<HTMLDivElement>});   
-    afterDomUpdate([], ()=>{
-        const {width, height} = state.ref.element!.getClientRects()[0];
-        state.area = width * height;
+    afterDomUpdate([], (consecutiveUpdatedFrames)=>{
```

Comment:
@nadavwix what do you think of this as a way to limit infinite loops potential?

Reply:
i'd have a maxConsecutive... as a third parameter to afterDomUpdate. and a default of 3, -1 (is infinite)
so by default it's limited